docs/conversion-tables/scss-lint.md

```diff
@@ -23,7 +23,7 @@ in the table.
 | --------------------------- | ----------------------- |
 | **[BangFormat][]** *enabled*<br>**`space_before_bang`**<br>`true` *default*<br>`false`<br><br>**`space_after_bang`**<br>`true`<br>`false` *default* | **[declaration-bang-space-before][] / [declaration-bang-space-after][]**<br>**`declaration-bang-space-before`**<br>`always`<br>`never`<br><br>**`declaration-bang-space-after`**<br>`always`<br>`never` |
 | **[BemDepth][]** *disabled*<br>**`max_elements`**<br>`number` *default: `1`* | - |
-| **[BorderZero][]** *enabled*<br>**`convention`**<br>`zero` *default*<br>`none` | - |
+| **[BorderZero][]** *enabled*<br>**`convention`**<br>`zero` *default*<br>`none` | **[value-border-zero][]**<br> `"convention": "0"`<br>`"convention": "none"`<br>(Plugin: [stylelint-value-border-zero][value-border-zero]) |
```

Comment:
This can be achieved using `stylelint`'s [`property-value-blacklist`](https://github.com/stylelint/stylelint/blob/master/src/rules/property-value-blacklist/README.md) rule i.e.

```js
{
  "property-value-blacklist": {
    "/^border/": ["none"] // or ["0"]
  }
}
```